generator/src/OperatorClassGenerator.php

```diff
@@ -76,11 +76,12 @@ public function createClass(GeneratorDefinition $definition, OperatorDefinition
                 $constuctor->setVariadic();
                 $constuctor->addComment('@param ' . $type->doc . ' ...$' . $argument->name . rtrim(' ' . $argument->description));
 
-                if ($argument->variadicMin !== null) {
+                if ($argument->variadicMin > 0) {
```

Comment:
Noted that ArgumentDefinition always sets this as an integer value, and `schema.json` likewise enforces only integer values if the config option is defined at all.